index.html

```diff
@@ -0,0 +1,459 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars" id="menu-button">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav" class="desk-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="menu hide" id="menu-id">
+      <button id="close">
+        <i>X</i>
+      </button>
+      <nav>
+        <ul>
+          <li><a href="#portfolio" class="menu-item">Portfolio</a></li>
+          <li><a href="#" class="menu-item">About</a></li>
+          <li class="last"><a href="#about-me" class="menu-item">Contact</a></li>
+        </ul>
+      </nav>
```

Comment:
- [ ]  Your effort is to salute, well-done @abdallahmalima 🥳 but we are afraid to say that you have used a bad `font-family` compared to this Figma [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=2yYSS3O1IFJjyQTJ-0) you have chosen, please just one more effort, update the `font-family` to match the correct template and you will be good to go ✌️
> | | Good font | | Bad font | |
> |-|---------|-|--------|-|
> | | ![mobile menu ref](https://user-images.githubusercontent.com/57408419/218095164-2a1fc0b9-1452-46d0-a060-dfcebe3afbab.jpg) | | ![mobile menu](https://user-images.githubusercontent.com/57408419/218095290-a1b78650-718e-4b91-83fb-be9bd6157998.jpg) | |